app/graphql/mutations/update_dynamic_field_category.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Mutations::UpdateDynamicFieldCategory < Mutations::BaseMutation
+  argument :id, Integer, required: true
+  argument :display_label, String, required: false
+  argument :sort_order, Integer, required: false
+
+  field :dynamic_field_category, Types::DynamicFieldCategoryType, null: true
+
+  def resolve(id:, **attributes)
+    dynamic_field_category = DynamicFieldCategory.find_by!(id: id)
```

Comment:
We might be able to use `find!` here! That's kinda exciting.

Reply:
It's so rare in Hyacinth, haha.